app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+###
+#  User class for authenticating via WebAuth
+###
+class User
+  attr_accessor :webauth
+
+  def initialize(webauth)
+    @webauth = webauth
+  end
+
+  def webauth_user?
+    @webauth.present?
```

Comment:
It seems to be needed by `<%= link_to "#{current_user.webauth}: Logout", logout_path(referrer: root_path) %>` in  _top_navbar.html.erb and tests